recipes-core/images/include/licenses.inc

```diff
@@ -10,5 +10,6 @@ tar_licenses() {
 	[ ! -e "${IMAGE_ROOTFS}/usr/share/common-licenses" ]
 }
 
-ROOTFS_POSTPROCESS_COMMAND += "tar_licenses; "
+# Fix this function for the BUILD_IMAGE_FROM_FEEDS case
```

Comment:
Could you reword this comment? I was confused the first time reading it. Something like
`# tar_licenses() doesn't work when using BUILD_IMAGE_FROM_FEEDS, because licenses are not available in the rootfs.`